src/libsystemd/sd-bus/test-bus-creds.c

```diff
@@ -2,6 +2,7 @@
 
 #include "sd-bus.h"
 
+#include "errno-util.h"
```

Comment:
sorting

Reply:
but it is already sorted?

(we usually put the external facing sd-*.h headers separately from the internal headers, if that#s what you find surprising here?)

Reply:
**errno**-util after **cgroup**-util, no?

Reply:
oh

Reply:
AI for the next conclave: sing the alphabet song all together